From 101d962ffa34c939911ac1a958c17cafaa4289e5 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Thu, 25 Oct 2018 18:20:16 -0700
Subject: [PATCH] Revert "fs/common: allow custom user table to override
 package-defined users"

This reverts commit f814dbf52270d50693bb9bbbcbfb700754efc771.

It would assign the different ID to the primary user, like bargee.
---
 fs/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/common.mk b/fs/common.mk
index b361ecce25..8ab85f5de9 100644
--- a/fs/common.mk
+++ b/fs/common.mk
@@ -53,10 +53,10 @@ rootfs-common: $(ROOTFS_COMMON_DEPENDENCIES) target-finalize
 	rm -rf $(FS_DIR)
 	mkdir -p $(FS_DIR)
 
-	$(call PRINTF,$(PACKAGES_USERS)) >> $(ROOTFS_FULL_USERS_TABLE)
 ifneq ($(ROOTFS_USERS_TABLES),)
 	cat $(ROOTFS_USERS_TABLES) >> $(ROOTFS_FULL_USERS_TABLE)
 endif
+	$(call PRINTF,$(PACKAGES_USERS)) >> $(ROOTFS_FULL_USERS_TABLE)
 
 	$(call PRINTF,$(PACKAGES_PERMISSIONS_TABLE)) > $(ROOTFS_FULL_DEVICES_TABLE)
 ifneq ($(ROOTFS_DEVICE_TABLES),)
-- 
2.20.1 (Apple Git-117)

